radio/src/telemetry/telemetry.cpp

```diff
@@ -562,12 +562,20 @@ int32_t convertTelemetryValue(int32_t value, uint8_t unit, uint8_t prec, uint8_t
   }
   else if (unit == UNIT_KTS) {
     if (destUnit == UNIT_KMH) {
-      // kts to km/h
+      // kts to km/h (1 knot = 1.85200 kilometers per hour)
       value = (value * 1852) / 1000;
     }
     else if (destUnit == UNIT_MPH) {
-      // kts to mph
-      value = (value * 23) / 20;
+      // kts to mph (1 knot = 1.15077945 miles per hour)
+      value = (value * 1151) / 1000;
```

Comment:
This one was already pretty good (23/20 = 1.15 ), but I decided to unify the calculation with the others.